spark_pipeline_framework/transformers/fhir_sender/v1/fhir_sender.py

```diff
@@ -437,6 +446,17 @@ def _transform(self, df: DataFrame) -> DataFrame:
                         ),
                     ).toDF(json_schema)
 
+                if debug:
+                    response_df = json_df.cache()
+                    response_df = response_df.coalesce(1)
+                    response_df.createOrReplaceTempView("result_view")
+
+                    FhirExporter(
+                        progress_logger=progress_logger,
+                        view="result_view",
+                        file_path=f"{file_path}/debug/response.json",
```

Comment:
file_path expects a directory since for each partition spark creates a separate file.